Source/DZNSegmentedControl.m

```diff
@@ -566,8 +566,8 @@ - (void)setSelectedSegmentIndex:(NSInteger)segment animated:(BOOL)animated
         
         [UIView animateWithDuration:duration
                               delay:0.0f
-             usingSpringWithDamping:damping
-              initialSpringVelocity:velocity
+//             usingSpringWithDamping:damping       // Antoine does not like the animation !
```

Comment:
My bad, i made a super quick and dirty fix and forgot a PR was pending on this branch